Changes.md

```diff
@@ -1,5 +1,10 @@
 # Sidekiq.cr Changelog
 
+0.7.0
```

Comment:
This alright?  Or should I add others too?

Reply:
I'm ok with 0.7.  You need to bump version.cr in tandem.

And correct the Crystal version below.